fire/testutils.py

```diff
@@ -58,21 +83,15 @@ def assertRaisesFireExit(self, code, regexp=None):
     """
     if regexp is None:
```

Comment:
as long as you're in here, this can be the default value to the fn.
`def assertRaisesFireExit(self, code, regexp='.*'):`